miflora-mqtt-daemon.py

```diff
@@ -5,6 +5,7 @@
 import json
 import os.path
 import argparse
+import time
 from time import sleep, localtime, strftime
```

Comment:
how about `from time import time, sleep, localtime, strftime` ?

Reply:
oh, didn't realize that there are already imports from time... will change that.